### PR TITLE
Fixed issue with type types from permissions rpc methods

### PR DESCRIPTION
**Pull request by shanejonas**
Had this odd issue with these tuple types when importing into mobile. This fixes it by explicitly stating the tuple types.

### CODE REVIEW

**Comment by shanejonas** [comment]
@metamaskbot publish-preview

**Review by Cal-L — DISMISSED** [review]
LGTM

File: packages/permission-controller/src/rpc-methods/index.ts [hunk 1]
```diff
@@ -7,4 +7,7 @@ import { getPermissionsHandler, GetPermissionsHooks } from './getPermissions';
 export type PermittedRpcMethodHooks = RequestPermissionsHooks &
   GetPermissionsHooks;
 
-export const handlers = [requestPermissionsHandler, getPermissionsHandler];
+export const handlers: [
+  typeof requestPermissionsHandler,
+  typeof getPermissionsHandler,
+] = [requestPermissionsHandler, getPermissionsHandler];
```
Comment (Mrtenz) [inline packages/permission-controller/src/rpc-methods/index.ts:13]:
```suggestion
export const handlers = [requestPermissionsHandler, getPermissionsHandler] as const;
```

**Review by Mrtenz — CHANGES_REQUESTED** [review]

**Comment by shanejonas** [comment]
@metamaskbot publish-preview

**Review by Mrtenz — APPROVED** [review]

**Review by BelfordZ — APPROVED** [review]

**Pull request merged**